docs/posts/arrow-flight-rpc-powered-by-ibis/arrow-flight-rpc-powered-by-ibis.qmd

```diff
@@ -0,0 +1,576 @@
+2024-02-06 14:53:15,422 - INFO     Connected to Flight Server at location: grpc+tls://localhost:8815
+2024-02-06 14:53:15,534 - INFO     Authenticated to the Flight Server as user: test
+2024-02-06 14:53:15,535 - INFO     Command Descriptor: <pyarrow.flight.FlightDescriptor cmd=b'{"command": "get_golden_rule_facts", "kwargs": {"num_endpoints": 1, "min_date": "1994-01-01T00:00:00", "max_date": "1995-12-31T00:00:00"}, "filters": [{"column": "o_custkey", "operator": "=", "value": 127171}], "columns": ["o_custkey", "o_orderkey"]}'>
+2024-02-06 14:53:17,129 - INFO        o_custkey  o_orderkey
+0     127171     2998659
+1     127171     2998659
+2     127171     3018402
+3     127171     3018402
+2024-02-06 14:53:17,131 - INFO        o_custkey  o_orderkey
+0     127171     4998663
+1     127171     4998663
+2     127171     4998663
+2024-02-06 14:53:17,132 - INFO     Got 7 rows total (56 bytes) - from 1 endpoint(s) (8 total chunk(s))
+2024-02-06 14:53:17,132 - INFO     Flight Client test: Elapsed time: 1.7104 seconds
+```
+
+In the output above, we only show 2 columns sent by the server, and further - we
+only retrieved 7 rows of data - for a total of 56 bytes!  We reduced the data
+sent by the server from 225,496,463 to 56 bytes - a significant savings in
+network throughput, and an effort in data processing offloaded to the server
+instead of the client. This is especially important in cases where a lightweight
+system must use the requested data, such as a FaaS platform or an embedded
+system.
+
+## Next steps
+
+In our subsequent blog posts on Arrow Flight RPC with Ibis, we’ll show:
+
+- How to install the example Flight RPC server with Ibis in Kubernetes - and
+ consuming it from your local laptop
+- How you can consume the Arrow Flight RPC server’s data from Spark - in
+ parallel for performance
+
+## Summary
+
+Here, we’ve showcased only one way you can use Arrow Flight RPC and Ibis to
+create powerful business APIs that exactly suit your needs efficiently and
+securely. In a composable data system, tools like Ibis and Arrow Flight RPC
+offer modularity that ensures you can remain flexible to business and technology
+changes at no cost to functionality. At Voltron Data, we focus on building
+systems like these, and would love to talk to you about how you can get started
+today.
+
+- For more information about Apache Arrow - see the
+ Arrow [homepage](https://arrow.apache.org/).
+- For more information about Apache Arrow Flight RPC -  see its
+ [documentation](https://arrow.apache.org/docs/format/Flight.html)
+- For more information about Ibis - see the Ibis
+ [homepage](https://ibis-project.org).
+
+A [Voltron Data Enterprise Support
+subscription](https://voltrondata.com/product) can help accelerate your success
```

Comment:
Since this is an open source project, ideally we can avoid the explicit selling here. We don't do this in other blog posts and I don't think we should start.

Reply:
we make other mentions of Voltron Data - should we remove those too?

Reply:
(removed the selling)